test/jdk/com/sun/jdi/CLETest.java

```diff
@@ -215,10 +215,15 @@ public void eventSetReceived(EventSet set) {
             // more than one Event in it.
             if (set.size() != 1) {
                 testcaseFailed = true;
-                // For now, we expect these two test cases to fail due to 8292217,
-                // so don't fail the overall test run as a result of these failures.
-                // testFailed = true;
-                System.out.println("TESTCASE #" + testcase + " FAILED (ignoring): too many events in EventSet: " + set.size());
+                // For now, we expect the 2nd test cases to fail due to 8292217,
+                // so don't fail the overall test run as a result of this failure.
+                // There is a workaround in place that allows the 1st test case to pass.
+                if (testcase == 1) {
+                    testFailed = true;
+                }
+                System.out.println("TESTCASE #" + testcase + " FAILED" +
+                                   (testcase == 2 ? "(ignoring)" : "") +
+                                   ": too many events in EventSet: " + set.size());
```

Comment:
testcase #2 is expected to fail, so that is why we do not set "testFailed = true" at line 222. However, if testcase #1 fails, we do want the overall test to fail, so in that case we seat "testFailed = true" at line 222.